addons/sourcemod/scripting/kento_mvp.sp

```diff
@@ -276,9 +314,31 @@ void DisplayMVPMenu(int client, int start)
 		Format(nomvp, sizeof(nomvp), "%T", "No MVP", client);
 		mvp_menu.AddItem("", nomvp);
 		
+		char steamid[32];
+		GetClientAuthId(client, AuthId_Engine, steamid, sizeof(steamid));
+		stock_ExtractSteamID(steamid, steamid, sizeof(steamid));
+		
 		for(int i = 1; i < MVPCount; i++)
 		{
-			mvp_menu.AddItem(g_sMVPName[i], g_sMVPName[i]);
+			if(g_hMVPSteamIds[i].Length) // If the MVP has any "restrictions"
+			{
+				if(g_hMVPSteamIds[i].FindString(steamid) != -1)
+				{
+					mvp_menu.AddItem(g_sMVPName[i], g_sMVPName[i]);
+				}
+				else
+				{
+					switch(Cvar_ShowMode.IntValue)
```

Comment:
I think hook cvar change and store value in a variable is better than calling `.IntValue` everytime.